user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md

```diff
@@ -148,6 +148,7 @@ It is not yet possible to configure time-to-live (TTL) values for STaaS. In the
 | Average trending (long)  | 10 years     |
 | State changes            | 5 years      |
 | Spectrum traces          | 1 year       |
+| Alarm events             | 1 year       |
```

Comment:
shouldn't we make the distinction between alarms and information events here?

Reply:
I just used the same table as it is used in the other docs:
https://docs.dataminer.services/user-guide/Advanced_Functionality/Databases/Specifying_TTL_overrides.html